kani-driver/src/call_single_file.rs

```diff
@@ -95,6 +100,14 @@ impl KaniSession {
         to_rustc_arg(vec![format!("--reachability={}", self.reachability_mode())])
     }
 
+    pub fn backend_arg(&self) -> Option<String> {
+        if self.args.common_args.unstable_features.contains(UnstableFeature::Aeneas) {
+            Some(to_rustc_arg(vec!["--backend=aeneas".into()]))
+        } else {
+            None
```

Comment:
Why do we return `None` here but `cprover` on line 59?

Reply:
Good catch. I updated the one on line 59.